www.foia.gov/_sass/theme/_typography.scss

```diff
@@ -18,7 +18,7 @@ h1 {
 }
 
 // Subhead 1
-h2 {
+%heading-2 {
```

Comment:
Whoa, what does the `%` do? I've never seen that!

Reply:
These are [placeholder selectors](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholders) or "template" selectors. They aren't rendered to CSS, but you can `@extend` them. My idea is to implement every CSS class in the WDS as placeholder classes, so you could extend them as you wish, or you could include uswds.css which only contains classes like:
```
.usa-grid {
  @extend %usa-grid;
}
```